x-pack/plugins/graph/public/components/workspace_layout/workspace_top_nav_menu.tsx

```diff
@@ -116,7 +125,14 @@ export const WorkspaceTopNavMenu = (props: WorkspaceTopNavMenuProps) => {
       defaultMessage: 'Inspect',
     }),
     run: () => {
-      props.setShowInspect((prevShowInspect) => !prevShowInspect);
+      onOpenInspector();
+    },
+    tooltip: () => {
+      if (isInspectDisabled) {
+        return i18n.translate('xpack.graph.topNavMenu.inspectButton.disabledTooltip', {
+          defaultMessage: 'Click Graph to enable Inspect',
```

Comment:
Also expanding can enable the inspector. What about something more general like `Perform a search or expand a node to enable Inspect` ?

Reply:
👍 for that!

Reply:
Changed